promscale/installation/recomm-guide.md

```diff
@@ -0,0 +1,39 @@
+# Resource recommendations
+This section describes the compute and disk requirements that are recommended for Promscale, based on the ingest rate and retention. 
+
+## Metrics
+Resource recommendation for ingestion through Prometheus remote-write. 
+
+### Prometheus remote write
+Compute recommendations for the Promscale connector and TimescaleDB are:
+
+| Ingestion Rate          | Connector CPU      | Connector Memory   | DB CPU | DB Memory| DB connections |
+| --------                | -------- | -------- | -------|----------|----------------|
+| 10k samples/sec      |     0.5     |      250 MB    |    1    |    4 GB      |          1      |
+| 50k samples/sec      |     2     |     700 MB     |    4    |     16 GB     |       2         |
+| 100k samples/sec   |    4      |     2 GB     |          8     |       32 GB         |  4|
+| 200k samples/sec   |     8     |     4.5 GB     |     16   |      64 GB    |        8        |
+| 500k samples/sec   |    32      |     9 GB     |     32   |     128 GB    |        64        |
```

Comment:
@VineethReddy02 @antekresic I don't think we were not able to achieve 500K sustained in cloud. I would remove if we are not sure we can achieve it.

Reply:
Yeah, until we reduce the IO load necessary, I don't think we will be able to support it on Cloud.